ui/admin/app/templates/scopes/scope/auth-methods/index.hbs

```diff
@@ -149,7 +149,6 @@
                     />
                     {{#unless B.data.isPrimary}}
                       <dd.Interactive
-                        data-test-make-primary-action
```

Comment:
Why this has been delete? [here context](https://github.com/hashicorp/boundary-ui/pull/2654/files#r1909599018)